UT4MasterServer/Controllers/ErrorsController.cs

```diff
@@ -1,29 +1,43 @@
 ﻿using Microsoft.AspNetCore.Diagnostics;
 using Microsoft.AspNetCore.Mvc;
 using UT4MasterServer.Exceptions;
+using UT4MasterServer.Models;
 
 namespace UT4MasterServer.Controllers;
 
 [ApiController, ApiExplorerSettings(IgnoreApi = true)]
 [Route("api/errors")]
 public class ErrorsController : ControllerBase
 {
+	private readonly ILogger<ErrorsController> logger;
+	public ErrorsController(ILogger<ErrorsController> logger)
+	{
+		this.logger = logger;
+	}
+
 	[HttpGet]
 	public IActionResult Index()
 	{
 		var message = "Internal server error occurred.";
 		var statusCode = 500;
 
 		var exceptionHandlerFeature = HttpContext.Features.Get<IExceptionHandlerPathFeature>();
+		var exception = exceptionHandlerFeature?.Error;
+
+		logger.LogError(exception, "Internal server error occurred.");
```

Comment:
We already have this string in `message` variable.
```suggestion
		logger.LogError(exception, message);
```

Reply:
I know, but VS is complaining about that:
![image](https://user-images.githubusercontent.com/12861786/212563569-1c3f6a7b-c251-46c5-bd39-7cfa9e5e0e6d.png)

I will store the text in a constant.